<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secure</class>
 <widget class="QDialog" name="secure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secure Settings</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="SaveSecureBtn">
      <property name="text">
       <string>Save to Secure</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancelSecureBtn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Identity Section</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Stamp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VersionEdt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SerialEdt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TimeStampEdt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CustomerEdt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>301</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Applications Section</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Version</string>
    </property>
    <widget class="QRadioButton" name="BasicRBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Basic</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="FullRbtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Full</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>141</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Apps</string>
    </property>
    <widget class="QCheckBox" name="ColorChkB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ProcessChkB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Process Control</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="IrradianceChkB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Irradiance</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="OxygenChkB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Oxygen</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ExcelChkB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Excel</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="PhotonChkB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Photon</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ThinFilmChkB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Thin Film</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ChemoChkB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Chemometry</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="FastTChkB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Fast Trigger</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Master/Slave</string>
    </property>
    <widget class="QRadioButton" name="NotDedRBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Dedicated</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="DedSlaveRBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dedicated Slave</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="DedMasterRBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dedicated Master</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Miscellaneous</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blackened Pixels Left</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blackened Pixels Right</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TEC Control P </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TEC Control I</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TEC Control D</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BlackLeftEdt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BlackRightEdt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TEC_P_Edt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TEC_I_Edt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TEC_D_Edt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MAC_Edt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DeviceTypeEdt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Nir Sensitivity</string>
   </property>
   <widget class="QRadioButton" name="NotSetRBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Set</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="LowNoiseRBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Low Noise</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HighSensRBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>High Sensitivity</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>VersionEdt</tabstop>
  <tabstop>SerialEdt</tabstop>
  <tabstop>TimeStampEdt</tabstop>
  <tabstop>CustomerEdt</tabstop>
  <tabstop>BasicRBtn</tabstop>
  <tabstop>FullRbtn</tabstop>
  <tabstop>NotDedRBtn</tabstop>
  <tabstop>DedSlaveRBtn</tabstop>
  <tabstop>DedMasterRBtn</tabstop>
  <tabstop>ColorChkB</tabstop>
  <tabstop>ProcessChkB</tabstop>
  <tabstop>IrradianceChkB</tabstop>
  <tabstop>OxygenChkB</tabstop>
  <tabstop>ExcelChkB</tabstop>
  <tabstop>PhotonChkB</tabstop>
  <tabstop>ThinFilmChkB</tabstop>
  <tabstop>ChemoChkB</tabstop>
  <tabstop>FastTChkB</tabstop>
  <tabstop>BlackLeftEdt</tabstop>
  <tabstop>BlackRightEdt</tabstop>
  <tabstop>TEC_P_Edt</tabstop>
  <tabstop>TEC_I_Edt</tabstop>
  <tabstop>TEC_D_Edt</tabstop>
  <tabstop>MAC_Edt</tabstop>
  <tabstop>DeviceTypeEdt</tabstop>
  <tabstop>NotSetRBtn</tabstop>
  <tabstop>LowNoiseRBtn</tabstop>
  <tabstop>HighSensRBtn</tabstop>
  <tabstop>SaveSecureBtn</tabstop>
  <tabstop>CancelSecureBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>SaveSecureBtn</sender>
   <signal>clicked()</signal>
   <receiver>secure</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelSecureBtn</sender>
   <signal>clicked()</signal>
   <receiver>secure</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
